File: dayOne/python/number_guesser.py
import random

number=0;
score=0;


randomNum= random.randint(1,100)
while True:
   
    
    user= int(input("guess the random number"))

    if user<randomNum:
        print("number is lower")
        
        score+=1

    elif user> randomNum:
        print("number is higer")
        
        score+=1

    elif user==randomNum:
        print("it is correct");
        score+=1
        break


    elif user> 100 or user<1:
        print(" ERROR")
        score+=1
    
    else:
        print("not a number");
        score+=1
    

    
    if score ==5:
        break;





if score==1:
    print(f"Final summary is correct number {randomNum} , attempts number: {score}")
    print("Rating: Lengendary")
elif score==2:

   print(f"Final summary is correct number {randomNum} , attempts number: {score}")
   print("Rating: Excellent")

elif score==3 or score ==4:
      print(f"Final summary is correct number {randomNum} , attempts number: {score}")
      print("Rating: Good")

elif score==5:

     print(f"Final summary is correct number {randomNum} , attempts number: {score}")
     print("Rating: Close")

else:
    print(f"Final summary is correct number {randomNum} , attempts number: {score}")
    print("Rating: Better Luck");

File: dayOne/java/NumberGuesser.java
import java.util.Random;
import java.util.Scanner;
public class NumberGuesser{



public static void main(String [] args){

    Scanner sc = new Scanner(System.in);
    Random random= new Random();
 
int randomNum= random.nextInt(101);
 int number=0;
int score=0;

System.out.print("guess the random number");
 randomNum= random.nextInt(101);

int user;
 while (number<6){

 

  user= sc.nextInt();



if (user<randomNum){
    System.out.println("number is lower");
   
   number++;

} else if (user> randomNum){
    System.out.println("number is higer");
  
  number++;
}
else if (user==randomNum){
    System.out.println("it is correct");
    score++;
    number++;
    break;
}

else if (user> 100|| user<1){
    System.out.println(" ERROR");
    
}
else{
 System.out.println("not a number");
 number++;

}

if (number==5){
    break;
}


System.out.println("guess the random number");


 




}

    if (number==1){
    System.out.printf("Final summary is correct number:%d , attempts number: %d%n, ", randomNum, number);
   System.out.print("Rating: Lengendary");
} else   if (number==2){

 System.out.printf("Final summary is correct number:%d , attempts number: %d%n, ", randomNum, number);
   System.out.print("Rating: Excellent");

} else   if (number==3|| number ==4){
 System.out.printf("Final summary is correct number:%d , attempts number: %d%n, ", randomNum, number);
   System.out.print("Rating: Good");

} else   if (number==5){

 System.out.printf("Final summary is correct number:%d , attempts number: %d%n, ", randomNum, number);
   System.out.print("Rating: Close");
}else {
    System.out.printf("Final summary is correct number:%d , attempts number: %d%n, ", randomNum, number);
   System.out.print("Rating: Better Luck");

}

















}
}
